src/mockServiceWorker.js

```diff
@@ -145,9 +145,9 @@ async function handleRequest(event, requestId) {
   // Ensure MSW is active and ready to handle the message, otherwise
   // this message will pend indefinitely.
   if (client && activeClientIds.has(client.id)) {
-    ;(async function () {
+    await (async function () {
       const clonedResponse = response.clone()
-      sendToClient(client, {
+      await sendToClient(client, {
```

Comment:
This is another example of a side effect which we don't need to await. In fact, I think it'd make request resolution slower if we do.

This line clones and sends back a response to the main thread. At this point, the request is finished, and we should really "release" it as soon as possible so the client reacts to its response. Anything that happens past this point is a side effect.